hal-base/hal/functions.py

```diff
@@ -494,14 +484,14 @@ def cleanNotifier(notifier):
 #############################################################################
 # PDP
 #############################################################################
-getPDPTemperature = _STATUSFUNC("getPDPTemperature", C.c_double)
```

Comment:
I've pushed a [changeset](https://usfirst.collab.net/gerrit/#/c/1085/) into gerrit to get this fixed, hopefully should get merged pretty quickly.